remove version variables and update docker note (#10)
hyperledger/besu-docs: docs/HowTo/Get-Started/Run-Docker-Image.md

docker run hyperledger/besu:latest
```

!!! note

>     Other tags are available for the image, see https://hub.docker.com/r/hyperledger/besu/tags

Reviewer: ```suggestion
    Available tags for the image are listed at https://hub.docker.com/r/hyperledger/besu/tags. 
```

>     Docker runs the cached version if you already pulled `latest`.

Reviewer: ```suggestion
   If you previously pulled `latest`, Docker runs the cached version.
```

>     Pull the `latest` version again using `docker pull hyperledger/besu:latest` to make sure your image is up to date. 

Reviewer: ```suggestion
    To ensure your image is up to date, pull the `latest` version again using `docker pull hyperledger/besu:latest`. 
```
